.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'bug: [description]'
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+**Relevant Logs**
+<!-- Setup `tracing_subscriber` in your application and use the `RUST_LOG` env variable to turn on logs. Please post any relevant logs here. Take a look at this guide on how to set up a tracing subscriber: https://tokio.rs/tokio/topics/tracing#subscribing-to-traces
```

Comment:
I find this description overly complex. In most cases just calling this will do, no? https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/fn.init.html

Reply:
I've had two cases of explaining people they need to install tracing subscriber to get logs this week.

Reply:
Ah - maybe I misinterpreted your comment. I'd rather have a link to `tracing_subscriber::fmt::init()` than a link to the "subscribing to traces" section in the tokio blog post.